reducer-todo-new/src/components/Todo.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const ToDo = props => {
+
+    const {dispatch} = props
+
+    return (
+        <div onClick = {() =>{ dispatch({ type : 'TOGGLE_ITEM', payload: props.item }) }}>
+            {props.item.completed ? 
+                <p style = {{textDecoration : 'line-through'}} className = 'list-item'>{props.item.task}</p>
+                : <p className = 'list-item'>{props.item.task}</p>
```

Comment:
instead of a ternary that returns 2 different elements, it might be cleaner to just have the ternary on the style.